charts/nfs-subdir-external-provisioner/values.yaml

```diff
@@ -108,3 +108,7 @@ affinity: {}
 
 # Additional labels for any resource created
 labels: {}
+
+podDisruptionBudget:
```

Comment:
values should be documented in the [chart readme](https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/blob/master/charts/nfs-subdir-external-provisioner/README.md#configuration)

Reply:
these have now been added